pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -163,7 +162,7 @@ type BundleCondition struct {
 	Type BundleConditionType `json:"type"`
 
 	// Status of the condition, one of ('True', 'False', 'Unknown').
-	Status corev1.ConditionStatus `json:"status"`
+	Status metav1.ConditionStatus `json:"status"`
```

Comment:
I think this was a mistake in the original code base.

Reply:
> I think this is a mistake in the original code base.

I can agree with this.